Mathlib/CategoryTheory/Monoidal/Rigid/Braided.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Gareth Ma. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gareth Ma
+-/
+import Mathlib.CategoryTheory.Monoidal.Rigid.Basic
+import Mathlib.CategoryTheory.Monoidal.Braided.Basic
+
+/-!
+# Mirror rigid structures via a braiding equivalence.
```

Comment:
Also what is "mirror"? Without reading the file below I don't think anyone would guess. The module doc should be interpretable in isolation. The module doc in particular should include `ExactPairing Y X` somewhere in the text!

Reply:
Looks good, thanks.